bitmap-team-awesome/test/color-constructor-test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// const expect = require('chai').expect;
+// const colorConstructor = require('../model/color-transform');
+
+// Need some help with how to properly test these functions
```

Comment:
@meeksfred let's make some time later this week/early next week to review this together - I'll help you determine what additional test components could be added in, as well as how to actually implement them.